packages/worker/src/transcribe.ts

```diff
@@ -172,27 +176,45 @@ const runTranscription = async (
 	}
 };
 
+const getLanguageCode = async (
+	whisperBaseParams: WhisperBaseParams,
+	whisperX: boolean,
+): Promise<LanguageCode> => {
+	if (whisperX) {
+		return Promise.resolve('auto');
```

Comment:
does whisperx not take a language arg?

Reply:
whisperx takes so long to start up I didn't think there was any point running a pre-pass of language detection and decided to just detect it every time